packages/amplify-e2e-tests/package.json

```diff
@@ -21,7 +21,8 @@
     "general-config-e2e": "jest src/__tests__/general-config/general-config-headless-init.test.ts --verbose",
     "build-tests": "tsc --build tsconfig.tests.json",
     "setup-profile": "ts-node ./src/configure_tests.ts",
-    "clean-e2e-resources": "ts-node ./src/cleanup-e2e-resources.ts"
+    "clean-e2e-resources": "ts-node ./src/cleanup-e2e-resources.ts",
+    "smoketest": "jest --runInBand --testPathPattern='smoketest'"
```

Comment:
should we have 
`yarn smoketest rc` and `yarn smoketest latest` ?
I.e. make them npm install `@latest` and `@rc` respectively?

Reply:
I like that idea. I'm working on a separate PR to integrate w/ CI, I'll get a PR up next week if time allows.